test/cri-containerd/jobcontainer_test.go

```diff
@@ -0,0 +1,87 @@
+// +build functional
+
+package cri_containerd
+
+import (
+	"context"
+	"strings"
+	"testing"
+	"time"
+
+	runtime "k8s.io/cri-api/pkg/apis/runtime/v1alpha2"
+)
+
+func getJobContainerPodRequestWCOW(t *testing.T) *runtime.RunPodSandboxRequest {
+	return &runtime.RunPodSandboxRequest{
+		Config: &runtime.PodSandboxConfig{
+			Metadata: &runtime.PodSandboxMetadata{
+				Name:      t.Name(),
+				Namespace: testNamespace,
+			},
+			Annotations: map[string]string{
+				"microsoft.com/hostprocess-container": "true",
+			},
+		},
+		RuntimeHandler: wcowProcessRuntimeHandler,
+	}
+}
+
+func getJobContainerRequestWCOW(t *testing.T, podID string, podConfig *runtime.PodSandboxConfig) *runtime.CreateContainerRequest {
+	return &runtime.CreateContainerRequest{
+		Config: &runtime.ContainerConfig{
+			Metadata: &runtime.ContainerMetadata{
+				Name: t.Name() + "-Container",
+			},
+			Image: &runtime.ImageSpec{
+				Image: imageWindowsNanoserver,
+			},
+			Command: []string{
+				"cmd",
+				"/c",
+				"ping",
+				"-t",
+				"127.0.0.1",
+			},
+
+			Annotations: map[string]string{
+				"microsoft.com/hostprocess":              "true",
+				"microsoft.com/hostprocess-inherit-user": "true",
+			},
+		},
+		PodSandboxId:  podID,
+		SandboxConfig: podConfig,
+	}
+}
+
+func Test_RunContainer_Username_JobContainer_WCOW(t *testing.T) {
```

Comment:
Are there more tests we can add that cover scenarios that will be expected to work? Doesn't have to be in this PR, but needs to come in ASAP if not here.

What are the most important privileged container scenarios?

Reply:
Doing networking configuration (HNS ops) seems to be a big one. We could invoke hnsdiag to satisfy a query example for now, or make another container image (when @anmaxvl's work gets in) that has a binary that does HNS configurations and then reverts them on exit. I think the ping example fills the role of validating "we can run things on the host now". 

Some good general functionality ones would be good like calling stats and having that work but that can be in the "coming in ASAP" basket.

Reply:
I can ask Mark/James if they want to see something in particular

Reply:
I was thinking something to make sure a container can call out to a privileged container would be good, or making sure a privileged container can install a program perhaps.

Reply:
Talking to them sounds good but let's please not frame this as "what they want" and rather make sure _we_ understand the customer requirements, and are testing adequately to cover them.

Reply:
As we are building more features that are used broadly, and trying to improve our quality, we should focus on making sure we are confident that if something important breaks we will be aware of it.

Reply:
(and the first step of that is understanding _what_ is important :) )

Reply:
Install a program sounds good, we'll just need to make sure it's reverted haha. Poor wording on my part so thanks for that, mainly meant are their any core scenarios for AKS or that they've heard of that would be nice to have here in addition to the k8s tests. 

And by calling out do you just mean launch a process isolated pod and have some workload container talk to a service the job container launched?

Reply:
I'm thinking 
1. HNS queries and general functionality. Validate we can create a dead simple network and remove after.
2. Look at disks/partitions/volumes on the host. 
3. Check hostname at beginning of test -> exec in job container and check hostname -> compare to validate that they're the same.
4.  Manipulating etw file (could have some image that houses https://github.com/microsoft/ETW2JSON)
5. Install (and uninstall on success) program

Reply:
> And by calling out do you just mean launch a process isolated pod and have some workload container talk to a service the job container launched?

Yes, if that's something users will care about. Would be good to test whatever communication method they intend to use (network, named pipe, etc).

Reply:
That list sounds good to me. If we have any use cases of customers collected, we should take a look at that to see if we're missing anything.

Reply:
We discussed and testing will be added in a (very soon) future PR.